src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props

```diff
@@ -28,6 +28,12 @@
     <NETCoreApp11Dependency Include="NETStandard.Library">
```

Comment:
Why didn't you update the runtime and host packages for 1.1? They should be up to 1.1.2